tests/indent/t32/if_block.cmm

```diff
@@ -43,7 +43,7 @@ IF &f
     stop
   ELSE
     IF &h
-  (
-    continue
-  )
+    (
+      continue
+    )
```

Comment:
I don't think this is a problem or anything.

As it stands, handling any if-else cases without specific `elseif` nodes, or without brackets will always be a mess. As long as it's consistent during usage (which to me seems like it does, as the changes to the test file here are very minimal).